compiler/fir/tree/src/org/jetbrains/kotlin/fir/BadNamedArgumentsTarget.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.fir
+
+enum class BadNamedArgumentsTarget(private val description:String) {
```

Comment:
Nit: I wonder IJ will warn `description:String` due to the lack of space after `:`?

Reply:
Interesting, auto format doesn't add space after the colon for me :S